fe/src/main/java/org/apache/doris/catalog/PartitionKey.java

```diff
@@ -120,11 +120,34 @@ public boolean isMinValue() {
 
     public boolean isMaxValue() {
         for (LiteralExpr literalExpr : keys) {
-            if (literalExpr != MaxLiteral.MAX_VALUE) {
-                return false;
+            if (literalExpr == MaxLiteral.MAX_VALUE) {
+                return true;
```

Comment:
Only all sub keys are MAX_VALUE that this partition key can be MaxValue.
partition key like `(MAX_VALUE, "100")` is not max value.

Reply:
You need to modify this

Reply:
ok